u8timeseries/tests/test_logging.py

```diff
@@ -79,21 +76,25 @@ def test_timeseries_split_error_log(self):
             get_logger('u8timeseries.timeseries').handlers = []
             try:
                 ts.split_after(pd.Timestamp('2020-02-01'))
-            except Exception:
+            except:
                 pass
-
+            
         lc.check(
             ('u8timeseries.timeseries', 'ERROR',
              'ValueError: Timestamp must be between 2000-01-01 00:00:00 and 2000-01-03 00:00:00')
         )
 
     def test_time_log(self):
-        # test time log decorator log message when running theta model
-        ts = constant_timeseries(length=100)
-        model = Theta()
+        logger = get_logger(__name__)
```

Comment:
Thanks, this is definitely a better test. But if we don't want the error log to show up when running unittest we should also add `logger.handlers = []`.

Reply:
thank you!